journals/apps/journals/views.py

```diff
@@ -106,6 +109,65 @@ def get(self, request, *args, **kwargs):
             })
 
 
+class JournalIndexView(WMABaseView):
```

Comment:
I would propose moving all Wagtail Admin related views into a separate file, like wagtailadmin_views.py. It's confusing having all views in one file when some are Wagtail Admin and some are Django Admin. I guess this applies to the Video views from @ziafazal as well.